cfme/common/__init__.py

```diff
@@ -286,10 +286,12 @@ def get_tags(self, tenant="My Company Tags"):
             List of tags in format "Tag_category: Tag_name"
         """
         view = navigate_to(self, 'Details')
-        tags = view.tag.read()
+        tags = view.smart_management.read()
```

Comment:
Please update class docs as 'tag' was changed to 'smart_management'. We need to get this clear, Details view should have 'smart_management', not 'tag'

Reply:
sure, changing..